tests/test_input.py

```diff
@@ -11,13 +11,7 @@
 import f90nml
 
```

Comment:
Tests are probably the easiest thing to read and the most sensible place to start (alongside the example notebook!)